app/controllers/members_controller.rb

```diff
@@ -4,5 +4,14 @@ def create
     event = Event.find(params[:event_id])
     event.members << current_user
     redirect_back(fallback_location: root_path)
-  end 
+  end
+
+  def destroy
+    event = Event.find(params[:event_id])
```

Comment:
Right now you have in two methods the same code: `event = Event.find(params[:event_id])` try to extract it to separate method. You have similar thing in events and places controller.

Reply:
Still valid.

Reply:
I have written code for it... hmm - now I see it on branch remove_event

```
  before_action :set_event, only: [:show, :edit, :update, :destroy]
```
```
def set_event
      @event = Event.find(params[:id])
    end 
```

Do you think about it?

Reply:
Yes this is what I meant, but in this controller you don't have action `:show, :edit` or `:update`. If I remember correctly you have `:create` and `:destroy`

Reply:
I have update it on branch button_remove_me

Reply:
Why not here? All changes with delete button should be one this branch or you create new one?